core/src/main/java/com/graphhopper/routing/CHPathCalculator.java

```diff
@@ -0,0 +1,91 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.graphhopper.routing;
+
+import com.graphhopper.routing.ch.CHRoutingAlgorithmFactory;
+import com.graphhopper.routing.querygraph.QueryGraph;
+import com.graphhopper.util.PMap;
+import com.graphhopper.util.Parameters;
+import com.graphhopper.util.StopWatch;
+
+import java.util.Collections;
+import java.util.List;
+
+import static com.graphhopper.util.EdgeIterator.ANY_EDGE;
+import static com.graphhopper.util.Parameters.Routing.MAX_VISITED_NODES;
+
+public class CHPathCalculator implements PathCalculator {
+    private final QueryGraph queryGraph;
+    private final CHRoutingAlgorithmFactory algoFactory;
+    private final PMap algoOpts;
+    private String debug;
+    private int visitedNodes;
+
+    public CHPathCalculator(QueryGraph queryGraph, CHRoutingAlgorithmFactory algoFactory, PMap algoOpts) {
+        this.queryGraph = queryGraph;
+        this.algoFactory = algoFactory;
+        this.algoOpts = algoOpts;
+    }
+
+    @Override
+    public List<Path> calcPaths(int from, int to, EdgeRestrictions edgeRestrictions) {
+        if (!edgeRestrictions.getUnfavoredEdges().isEmpty())
+            throw new IllegalArgumentException("Using unfavored edges is currently not supported for CH");
+        RoutingAlgorithm algo = createAlgo();
+        return calcPaths(from, to, edgeRestrictions, algo);
+    }
+
+    private RoutingAlgorithm createAlgo() {
+        StopWatch sw = new StopWatch().start();
+        RoutingAlgorithm algo = algoFactory.createAlgo(queryGraph, algoOpts);
+        debug = ", algoInit:" + (sw.stop().getNanos() / 1000) + " micros";
```

Comment:
I know you only copied this, but we should make it consistent. E.g. in Measurement we use `us` instead of `micros`. Or even `µs` :) ?

Reply:
I like using the actual greek letter, I just need to figure out how to type it on my keyboard :) There shouldn't be any problem using it?